### PR TITLE
feat: Adding leaderboard for multilingual

### PR DESCRIPTION
**Pull request by Ciroye**

### CODE REVIEW

File: backend/app/domain/services/base/model.py [hunk 1]
```diff
@@ -489,3 +490,8 @@ def amount_of_models_uploaded_in_hr_diff(self, task_id: int, user_id: int):
             )
         )
         return amount_of_models_uploaded_in_hr_diff
+
+    def get_dynalab_model(self, task_code: str):
+        bucket = "https://models-dynalab.s3.eu-west-3.amazonaws.com"
```
Comment (remg1997) [inline backend/app/domain/services/base/model.py:495]: I'm sure there's a field in the database, in the task table, for the bucket variable.

File: backend/app/domain/services/builder_and_evaluation/eval_utils/input_formatter.py [hunk 2]
```diff
@@ -91,6 +91,21 @@ def group_labels(examples: list) -> dict:
     return final_labels
 
 
+def neccesary_format_for_evaluation_multilingual(
```
Comment (remg1997) [inline backend/app/domain/services/builder_and_evaluation/eval_utils/input_formatter.py:94]:
```suggestion
def necessary_format_for_multilingual_evaluation(
```

File: backend/app/domain/services/builder_and_evaluation/eval_utils/input_formatter.py [hunk 3]
```diff
@@ -91,6 +91,21 @@ def group_labels(examples: list) -> dict:
...
+    ]
+    list_of_predictions = [
+        entry[label] for entry in load_dataset(prediction_dict["predictions"])
+    ]
+    formated_dict["formatted_base_predictions"] = list_of_predictions
+    formated_dict["formatted_base_dataset"] = list_of_labels
+    return formated_dict, False
+
+
 def neccesary_format_for_evaluation(prediction_dict: dict, label: str):
```
Comment (remg1997) [inline backend/app/domain/services/builder_and_evaluation/eval_utils/input_formatter.py:109]:
```suggestion
def necessary_format_for_evaluation(prediction_dict: dict, label: str):
```

File: backend/app/domain/services/builder_and_evaluation/eval_utils/metrics.py [hunk 4]
```diff
@@ -157,6 +157,15 @@ def get_dataperf_balanced_accuracy_meta(task=None):
     }
 
 
+def get_chrf_meta(task=None):
+    return {
+        "unit": "%",
+        "pretty_name": "Balanced Accuracy",
```
Comment (remg1997) [inline backend/app/domain/services/builder_and_evaluation/eval_utils/metrics.py:163]: We need to change the pretty name for either chrF or Character n-gram F-score

File: frontends/web/src/new_front/components/Buttons/CreateInterfaceHelpersButton.tsx [hunk 5]
```diff
@@ -19,12 +19,16 @@ const CreateInterfaceHelpersButton: FC<CreateInterfaceHelpersButtonProps> = ({
...
   return (
     <div className="flex items-center justify-between gap-2">
-      {amountsExamplesCreatedToday !== null ? (
-        <div className="text-xs text-gray-500">
-          {amountsExamplesCreatedToday} examples created
-        </div>
-      ) : null}
-      <ShowInstructionsButton />
+      <div className="col-span-2 ">
+        {amountsExamplesCreatedToday !== null ? (
```
Comment (remg1997) [inline frontends/web/src/new_front/components/Buttons/CreateInterfaceHelpersButton.tsx:23]:
```suggestion
        {amountExamplesCreatedToday !== null ? (
```

File: frontends/web/src/new_front/components/Buttons/CreateInterfaceHelpersButton.tsx [hunk 6]
```diff
@@ -19,12 +19,16 @@ const CreateInterfaceHelpersButton: FC<CreateInterfaceHelpersButtonProps> = ({
...
+          <div className="text-xs text-gray-500">
+            {amountsExamplesCreatedToday} examples created
```
Comment (remg1997) [inline frontends/web/src/new_front/components/Buttons/CreateInterfaceHelpersButton.tsx:25]:
```suggestion
            {amountExamplesCreatedToday} examples created
```

File: frontends/web/src/new_front/components/Tables/Leaderboard/PairLanguageLeaderboard.tsx [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+    "en-zh",
+    "en-tr",
+    "en-ar",
+    "en-ja",
+    "en-ko",
+    "en-pt",
+    "en-it",
+  ];
+
+  const resultModels = [
```
Comment (remg1997) [inline frontends/web/src/new_front/components/Tables/Leaderboard/PairLanguageLeaderboard.tsx:23]: What are this constants for?

**Review by remg1997 — CHANGES_REQUESTED** [review]
Small changes

File: backend/app/domain/services/base/model.py (hunk 1 above)
Comment (Ciroye) [reply backend/app/domain/services/base/model.py:495]: Not really for the dynalab model

File: frontends/web/src/new_front/components/Tables/Leaderboard/PairLanguageLeaderboard.tsx (hunk 7 above)
Comment (Ciroye) [reply frontends/web/src/new_front/components/Tables/Leaderboard/PairLanguageLeaderboard.tsx:23]: This is mock data, in a near nearr future (this weekend) will be solved

Comment (remg1997) [reply frontends/web/src/new_front/components/Tables/Leaderboard/PairLanguageLeaderboard.tsx:23]: Sounds good

**Review by remg1997 — APPROVED** [review]

**Pull request merged**